<!-- <?xml version="1.0" encoding="UTFâˆ’8"?>
<launch>
	<include file="$(find gazebo_ros)/launch/empty_world.launch"/>
	<include file="$(find kobuki_gazebo)/launch/kobuki_empty_world.launch"/>
	<node name="A" pkg="week2_170040012_190100057" type="A.py" output="screen"/>
	<node name="B" pkg="week2_170040012_190100057" type="B.py" output="screen"/>
	<node name="C" pkg="week2_170040012_190100057" type="C.py" output="screen"/>
</launch>
 -->
<!-- <?xml version="1.0" encoding="UTF-8"?> -->
<launch>
  <include file="$(find gazebo_ros)/launch/empty_world.launch"/>
  	<!-- <arg name="world_name" value="$(find week3)/worlds/landmarks.world"/> -->
  <!-- </include> -->

  <include file="$(find kobuki_gazebo)/launch/includes/robot.launch.xml"/>

  <node name="A" pkg="week2_170040012_190100057" type="A.py" output="screen"/>
  <node name="B" pkg="week2_170040012_190100057" type="B.py" output="screen"/>
  <node name="C" pkg="week2_170040012_190100057" type="C.py" output="screen"/>
  <!-- <node name="A" pkg="week2_170040012_190100057" type="A.py" output="screen"/> -->
</launch>

